src/webui/webapplication.cpp

```diff
@@ -265,10 +265,6 @@ void WebApplication::translateDocument(QString &data) const
             // it should fallback to `sourceText`
             QString translation = loadedText.isEmpty() ? sourceText : loadedText;
 
-            // Use HTML code for quotes to prevent issues with JS
```

Comment:
Still, I wanted to note that this comment is misleading.
The need of escaping depends on the usage site. With proper code, JS shouldn't need quote escaping.
AFAIK only html attributes would benefit from escaping and even so it is not always required. For example, browser would accept this: `<p data-xyz="'"></p>`. Other usage in HTML shouldn't need quote escaping.

Given this, escaping every string just to prevent html attributes go wrong isn't the correct way. It is a workaround at best and not even a good one.

https://stackoverflow.com/a/7382028